content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -202,49 +159,49 @@ that looks something like this (output is line-wrapped for readability):
 <summary>Traces and Logs</summary>
 
 ```log
-LogRecord.Timestamp:               2023-08-14T06:44:53.9279186Z
-LogRecord.TraceId:                 3961d22b5f90bf7662ad4933318743fe
-LogRecord.SpanId:                  93d5fcea422ff0ac
+ogRecord.Timestamp:               2023-10-23T12:13:30.2704325Z
+LogRecord.TraceId:                 324333ec3bbca04ba7f4be4bf3618cb1
+LogRecord.SpanId:                  e7d3814e31e504eb
 LogRecord.TraceFlags:              Recorded
-LogRecord.CategoryName:            simple-dotnet
-LogRecord.LogLevel:                Information
-LogRecord.StateValues (Key:Value):
+LogRecord.CategoryName:            Program
+LogRecord.Severity:                Info
+LogRecord.SeverityText:            Information
+LogRecord.Body:                    Anonymous player is rolling the dice: {result}
+LogRecord.Attributes (Key:Value):
     result: 1
     OriginalFormat (a.k.a Body): Anonymous player is rolling the dice: {result}
 
 Resource associated with LogRecord:
-service.name: simple-dotnet
-telemetry.auto.version: 0.7.0
+service.name: roll-dice
+service.instance.id: f20134f3-293f-4cb2-ace3-724b5571ca9a
 telemetry.sdk.name: opentelemetry
 telemetry.sdk.language: dotnet
-telemetry.sdk.version: 1.4.0.802
+telemetry.sdk.version: 1.6.0
 
-info: simple-dotnet[0]
-      Anonymous player is rolling the dice: 1
-Activity.TraceId:            3961d22b5f90bf7662ad4933318743fe
-Activity.SpanId:             93d5fcea422ff0ac
+Activity.TraceId:            324333ec3bbca04ba7f4be4bf3618cb1
+Activity.SpanId:             e7d3814e31e504eb
 Activity.TraceFlags:         Recorded
-Activity.ActivitySourceName: OpenTelemetry.Instrumentation.AspNetCore
+Activity.ActivitySourceName: Microsoft.AspNetCore
 Activity.DisplayName:        /rolldice
 Activity.Kind:               Server
-Activity.StartTime:          2023-08-14T06:44:53.9278162Z
-Activity.Duration:           00:00:00.0049754
+Activity.StartTime:          2023-10-23T12:13:30.2163005Z
+Activity.Duration:           00:00:00.0585187
 Activity.Tags:
-    net.host.name: localhost
-    net.host.port: 8080
+    net.host.name: 127.0.0.1
+    net.host.port: 5197
     http.method: GET
     http.scheme: http
     http.target: /rolldice
-    http.url: http://localhost:8080/rolldice
+    http.url: http://127.0.0.1:5197/rolldice
```

Comment:
```suggestion
    http.url: http://127.0.0.1:8080/rolldice
```

Please make sure that the sample app runs on the same port as suggested above (8080 to be consistent across language docs)